app/views/rubygems/_gem_members.html.erb

```diff
@@ -4,6 +4,17 @@
     <div class="gem__users">
       <%= links_to_owners(rubygem) %>
     </div>
+
+    <% if rubygem.owned_by?(current_user) && rubygem.owners.without_mfa.present? %>
+      <span class="gem__users__mfa-text"><%= t '.not_using_mfa_warning_show' %></span>
+
+      <div class="gem__users__mfa-disabled t-item--hidden">
+        <span class="gem__users__mfa-text t-item--hidden"><%= t '.not_using_mfa_warning_hide' %></span>
+        <div class="gem__users">
+          <%= links_to_owners_without_mfa(rubygem) %>
+        </div>
+      </div>
+    <% end %>
```

Comment:
Consider showing a message along the lines of "All owners have MFA enabled" when that's the case.

Reply:
fixed in 53438a9